src/time_integration/paired_explicit_runge_kutta/polynomial_optimizer.jl

```diff
@@ -0,0 +1,28 @@
+# Filter out eigenvalues with positive real parts, those with negative imaginary
+# parts due to eigenvalues' symmetry around the real axis, or the eigenvalues
+# that are smaller than a specified threshold.
+function filter_eig_vals(eig_vals, threshold = 1e-12; verbose = false)
+    filtered_eig_vals = Complex{Float64}[]
+
+    for eig_val in eig_vals
```

Comment:
We may need to re-think this logic when using Triix.jl with `Float32`. Do you already have experience with this case? If not, we may just add a comment here or a TODO note to our `Float32` issue.

Reply:
We explicitly use `Complex{Float64}` instead of `ComplexF64` to have easier type-exchange. 
Since this is essentially pre-processing, I am not sure if we need the conversion to `Float32` here.